docker-compose.dev.yml

```diff
@@ -0,0 +1,106 @@
+version: "3.7"
```

Comment:
Please make sure you do git mv, instead of creating a new file, otherwise we lost historial changes

Reply:
That's what I did and what my local git is saying. Probably an issue because the old name of one file is the same as the new name of the other

Reply:
I think you will need to revert this and then do:
git mv docker-compose.yml docker-compose.dev.yml
git mv docker-compose.latest.yml docker-compose.yml